modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectNotFoundEntriesManagementToolbarDisplayContext.java

```diff
@@ -204,6 +204,14 @@ public String getDefaultEventHandler() {
 		).build();
 	}
 
+	public PortletURL getIteratorURL() {
```

Comment:
@AliciaGarciaGarcia I think there is a better way to fix this, we shouldn't need to add the navigation parameter. I think the problem is in the `RedirectNotFoundEntriesDisplayContext._getPorteltURL` method that should return a clone of the current URL to include all the parameters, and not only the navigation. 

I am not 100% sure though, we can discuss it tomorrow.

Reply:
See https://github.com/liferay-lima/liferay-portal/pull/29.